src/api/methods/index.js

```diff
@@ -57,6 +57,7 @@ module.exports = (bridgeState, db, app, tendermintPort) => {
       null,
       bridgeState
     ),
+    plasma_getState: async () => bridgeState,
```

Comment:
For consistency sake, maybe it's worth to put it into a separate file?
https://github.com/leapdao/leap-node/blob/master/src/api/methods/getBlockNumber.js is sorta a one-liner as well :)